packages/dds/tree/src/test/shared-tree-core/edit-manager/editManagerScenario.ts

```diff
@@ -45,11 +41,17 @@ interface UnitTestAckStep {
 	type: "Ack";
 	/**
 	 * The sequence number for this change.
-	 * Should match the sequence number of the oldest `UnitTestPushStep`
-	 * for which there is no `UnitTestAckStep` step.
+	 * Should match the sequence number of the oldest `UnitTestPushStep` for which there is no `UnitTestAckStep` step.
+	 * Basically, a `Push` step and its corresponding `Ack` step must have the same sequence number.
 	 */
 	seq: number;
 }
+/**
+ * The intention property represents the change associated with the step. The ack step will have the intention value for
+ * the push-ack pair. Note that in cases where there are more than one step with the same sequence number, the intention
```

Comment:
I don't understand this sentence: "The ack step will have the intention value for the push-ack pair." Did you mean "the same intention value as the push"?

Reply:
Updated the comment, please take a look now. Basically, the idea is that a push-ack pair represent the same change / commit and so, will have the same intention value.